src/Fabulous.XamarinForms/Views/Layouts/FlexLayout.fs

```diff
@@ -0,0 +1,128 @@
+        Attributes.defineBindable<FlexDirection> FlexLayout.DirectionProperty
+
+    let Grow =
+        Attributes.defineBindable<float> FlexLayout.GrowProperty
+
+    let JustifyContent =
+        Attributes.defineBindable<FlexJustify> FlexLayout.JustifyContentProperty
+
+    let Order =
+        Attributes.defineBindable<int> FlexLayout.OrderProperty
+
+    let Position =
+        Attributes.defineBindable<FlexPosition> FlexLayout.PositionProperty
+
+    let Shrink =
+        Attributes.defineBindable<float> FlexLayout.ShrinkProperty
+
+    let Wrap =
+        Attributes.defineBindable<FlexWrap> FlexLayout.WrapProperty
+
+
+[<AutoOpen>]
+module FlexLayoutBuilders =
+    type Fabulous.XamarinForms.View with
+        static member inline FlexLayout<'msg>() =
+            CollectionBuilder<'msg, IFlexLayout, IView>(FlexLayout.WidgetKey, LayoutOfView.Children)
+
+[<Extension>]
+type FlexLayoutModifiers =
+    /// <summary>Sets a value that controls how multiple rows or columns of child elements are aligned.</summary>
+    /// <param name="value">Enumerates values that control how multiple rows or columns of child elements are aligned.</param>
+    [<Extension>]
+    static member inline alignContent(this: WidgetBuilder<'msg, #IFlexLayout>, value: FlexAlignContent) =
+        this.AddScalar(FlexLayout.AlignContent.WithValue(value))
+
+    /// <summary>Sets a value that controls how child elements are laid out within their row or column.</summary>
+    /// <param name="value">Enumerates values that control the alignment of child elements.</param>
+    [<Extension>]
+    static member inline alignItems(this: WidgetBuilder<'msg, #IFlexLayout>, value: FlexAlignItems) =
+        this.AddScalar(FlexLayout.AlignItems.WithValue(value))
+
+    /// <summary>Sets the flex direction for child elements within this layout.</summary>
+    /// <param name="value">Enumerates values that specify row and colum in flex layout directions, relative to the directions for the device locale.</param>
+    [<Extension>]
+    static member inline direction(this: WidgetBuilder<'msg, #IFlexLayout>, value: FlexDirection) =
+        this.AddScalar(FlexLayout.Direction.WithValue(value))
+
+    /// <summary>Sets a value that controls whether and how child elements within this layout wrap.</summary>
+    /// <param name="value">Enumerates values that control whether and how to wrap items in a FlexLayout.</param>
+    [<Extension>]
+    static member inline wrap(this: WidgetBuilder<'msg, #IFlexLayout>, value: FlexWrap) =
+        this.AddScalar(FlexLayout.Wrap.WithValue(value))
+
+    /// <summary>Sets a value that controls whether the coordinates of child elements are specified in absolute or relative terms.</summary>
+    /// <param name="value">Enumerates values that control how layout coordinates are interpreted when specifying the positions of child elements.</param>
+    [<Extension>]
+    static member inline position(this: WidgetBuilder<'msg, #IView>, value: FlexPosition) =
```

Comment:
I think both position and justify content should be constrained to `#IFlexLayout`